PyInstaller/hooks/hook-shapely.py

```diff
@@ -9,21 +9,27 @@
 
 import os
 
-from PyInstaller.utils.hooks import get_package_paths
-from PyInstaller.utils.hooks import is_module_satisfies
-from PyInstaller import compat
+from PyInstaller.utils.hooks import get_package_paths, is_module_satisfies
```

Comment:
Please abstain from *any* change not necessary, even if the code does not strictly follow PEP 8. The benefit is too small compared to the efforts when we need to reconstruct a change-history. Thanks.

Reply:
Not sure I completely agree with this, but ok. If you don't do this how/when do you fix ugly code when you notice it?

Reply:
We just live with the ugly code (as long as it is acceptable) and try to avoid new ugly code :-)

Too often I found myself walking through the history, trying to spot relevant changes and wasting time on "reformatting" changes. See 2727 for a lengthy discussion.